src/annotations/core/AnnotationArea.cpp

```diff
@@ -309,6 +309,9 @@ void AnnotationArea::deleteSelectedItems()
 	auto selectedItems = mItemModifier->selectedItems();
 	mItemModifier->clear();
 	mUndoStack->push(new DeleteCommand(selectedItems, this));
+	if (mSettingsProvider->toolType() == Tools::Select) {
+		mSettingsProvider->activateSelectTool();
```

Comment:
We only want to remove the properties of the previously selected item. Re-activating the select tool will do so.

Reply:
Is there a case where you have a different tool then select after deleting an item? I think in order to delete an item you have to select it either by triggering the context menu or selecting it with a selection region, both cases will select the select tool so you are basically always running into this if. I might be wrong but can't think of any use case at the moment.

Reply:
@DamirPorobic Yes, you can have a different tool after deleting an item. You can do the following:

1. select selection tool
2. select item to delete
3. select another tool (item remains selected)
4. delete item

Reply:
Nice catch, didn't think about that.